src/controllers/player.coffee

```diff
@@ -159,13 +176,31 @@ Matching.controller 'matchingPlayerCtrl', ['$scope', '$timeout', '$sce', ($scope
 				$scope.pageAnimate = false
 			, ANIMATION_DURATION*1.1
 
+		if _boardElement then _boardElement.focus()										 
+		if direction == 'next' then _assistiveAlert 'Page incremented. You are on the next page.' 
+		else if direction == 'previous' then _assistiveAlert 'Page decremented. You are on the previous page.'
+
+
+	_updateCompletionStatus = () ->
```

Comment:
This function appears to be causing the "___ of 5 remaining" issue. It resets the `completePerPage` array before refilling it using the matches each time a match is made. It would need another loop that finds pages with no matches, and sets those pages' counts to 0 in the `completePerPage` array. 

The more efficient method may be to just replace the call to `_updateCompletionStatus` with `$scope.completePerPage[$scope.currentPage]++` inside `_pushMatch`.

Reply:
Thank you!